src/modal-create-component.js

```diff
@@ -62,19 +74,28 @@ function addIcons () {
 
     let iconsDescription = document.createElement('label');
     iconsDescription.classList.add('pv-label');
-    iconsDescription.textContent = 'Choose if it will have an icon:';
+    iconsDescription.textContent = 'Select an icon:';
 
     let iconsList = document.createElement('div');
     iconsList.classList.add('inline-block', 'pv-icons');
 
-    _octicons.icons.forEach(
-        (icon) => {
-            let entryIcon = document.createElement('button');
-            entryIcon.classList.add('btn', 'btn-sm', 'inline-block-tight', 'text-subtle', 'icon', icon);
-            entryIcon.textContent = icon;
-            entryIcon.addEventListener('click', addIconClickEvent.bind(this), false);
-            iconsList.appendChild(entryIcon);
+    let loopIcons = ( function (icon) {
```

Comment:
THOUGHT: can we make this as another function instead on an IIFE? and then call as `loopIcons.call(this, icon)`? what's your opinion?